### PR TITLE
Add German Translations to GUI

### PR DESCRIPTION
**Pull request by Philip6294**

### CODE REVIEW

File: gui/src/i18n/de/translation.json [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+      "title": "Tracker zuweisen",
+      "description": "Lassen Sie uns auswählen, welcher Tracker wo hin geht. Klicken Sie auf einen Ort, an dem Sie einen Tracker platzieren möchten",
+      "assigned": "{{assigned}} von {{trackers}} Trackern zugewiesen",
+      "advanced": "Erweiterte Zuweisungsorte anzeigen",
+      "next": "Ich habe alle Tracker zugewiesen"
+    },
+    "manual-mounting": {
+      "back": "Zurück zum Eintritt in die VR",
+      "title": "Manuelles Befestigen",
+      "description": "Klicken Sie auf jeden Tracker und wählen Sie aus, in welche Richtung sie montiert sind",
```
Comment (TheDevMinerTV) [inline gui/src/i18n/de/translation.json:368]: sie -> diese

File: gui/src/i18n/de/translation.json [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+      "auto-mounting": "Automatisches Befestigen",
+      "next": "Nächster Schritt"
+    },
+    "automatic-mounting": {
+      "back": "Zurück zum Eintritt in die VR",
```
Comment (TheDevMinerTV) [inline gui/src/i18n/de/translation.json:373]: Zurück zum Eintritt in VR

File: gui/src/i18n/de/translation.json [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+      "prev-step": "Vorheriger Schritt",
+      "done": {
+        "title": "Befestigungsdrehungen kalibriert.",
+        "description": "Ihre Befestigungskalibrierung ist abgeschlossen!",
+        "restart": "Zurück zum Start"
+      },
+      "mounting-reset": {
+        "title": "Befestigung zurücksetzen",
+        "step": {
+          "0": "1. Beugen Sie sich im \"Skifahren\"-Pose mit gebeugten Beinen, geneigtem Oberkörper und gebeugten Armen.",
```
Comment (TheDevMinerTV) [inline gui/src/i18n/de/translation.json:387]: im -> in die

File: gui/src/i18n/de/translation.json [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+      },
+      "preparation": {
+        "title": "Vorbereitung",
+        "step": {
+          "0": "1. Stehen Sie aufrecht mit Ihren Armen an den Seiten.",
+          "1": "2. Drücken Sie die Schaltfläche \"Zurücksetzen\" und warten Sie 3 Sekunden, bevor die Tracker zurückgesetzt werden."
+        }
+      },
+      "put-trackers-on": {
+        "title": "Ziehen sie ihre Tracker an",
```
Comment (TheDevMinerTV) [inline gui/src/i18n/de/translation.json:399]: Sie

File: gui/src/i18n/de/translation.json [hunk 5]
```diff
@@ -0,0 +1,462 @@
...
+    "manual-proportions": {
+      "back": "Gehen Sie zurück zum Reset-Tutorial",
+      "title": "Manuelle Körperproportionen",
+      "precision": "Fein-Anpassung",
+      "auto": "Automatische Kalibrierung"
+    },
+    "automatic-proportions": {
+      "back": "Gehen Sie zurück zum Reset-Tutorial",
+      "title": "Messen Sie Ihre Proportionen",
+      "description": "Damit die SlimeVR-Tracker funktionieren, müssen wir ihre Proportionen kennen. Diese kurze Kalibrierung wird sie für Sie messen.",
```
Comment (TheDevMinerTV) [inline gui/src/i18n/de/translation.json:413]: Ihre

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: gui/src/i18n/de/translation.json [hunk 6]
```diff
@@ -0,0 +1,462 @@
...
+    "new-device.p1": "Geben Sie Ihre WLAN-Anmeldeinformationen ein!",
+    "new-device.p2": "Bitte wählen Sie, was Sie damit machen möchten",
+    "open-wifi": "Mit WLAN verbinden",
+    "open-serial": "Seriellen Konsole öffnen",
+    "submit": "Absenden!",
+    "close": "Schließen"
+  },
+  "navbar": {
+    "home": "Start",
+    "body-proportions": "Körper-proportionen",
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:71]: Yes, but if it's spelled together the text will clip (see Conversation in Forum Post, Moof suggested this). I would life it spelled with a dash

File: gui/src/i18n/de/translation.json [hunk 7]
```diff
@@ -0,0 +1,462 @@
...
+      },
+      "mounting-section": {
+        "title": "Befestigungsposition",
+        "description": "Wo ist der Tracker befestigt?",
+        "edit": "Befestigung bearbeiten"
+      },
+      "name-section": {
+        "title": "Trackername",
+        "description": "Geben Sie ihm einen süßen Spitznamen :)",
+        "input-placeholder": "NightyBeast's linke Bein"
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:133]: ( I'm tempted to write something like 'Hinkebein' here lol)

File: gui/src/i18n/de/translation.json [hunk 8]
```diff
@@ -0,0 +1,462 @@
...
+    "FOOT_LENGTH": "Fußlänge",
+    "FOOT_SHIFT": "Fußverschiebung",
+    "SKELETON_OFFSET": "Skelettversatz",
+    "CONTROLLER_DISTANCE_Z": "Controllerabstand Z",
+    "CONTROLLER_DISTANCE_Y": "Controller Entfernung Y",
+    "FOREARM_LENGTH": "Unterarm Entfernung",
+    "SHOULDERS_DISTANCE": "Schulter Entfernung",
+    "SHOULDERS_WIDTH": "Schulter Breite",
+    "UPPER_ARM_LENGTH": "Oberarm Länge",
+    "ELBOW_OFFSET": "Ellbogen Versatz"
```
Comment (TheDevMinerTV) [inline gui/src/i18n/de/translation.json:52]: Not sure if these should be written as one word or not, but I believe they should be

**Review by carl-anders — APPROVED** [review]

File: gui/src/i18n/de/translation.json (hunk 8 above)
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:52]: Does not really seem to matter if it's one word or not but I'll change it to one

File: gui/src/i18n/de/translation.json [hunk 9]
```diff
@@ -0,0 +1,462 @@
...
+  },
+  "reset": {
+    "reset-all": "Alle Proportionen zurücksetzen",
+    "full": "Reset",
+    "mounting": "Befestigungs-Reset",
+    "quick": "Schneller Reset"
+  },
+  "serial-detection": {
+    "new-device.p0": "Neues serielles Gerät erkannt!",
+    "new-device.p1": "Geben Sie Ihre WLAN-Anmeldeinformationen ein!",
```
Comment (unlogisch04) [inline gui/src/i18n/de/translation.json:62]:
```suggestion
    "new-device.p1": "Geben Sie Ihre WLAN-Zugangsdaten ein!",
```
Using "Zugangsdaten"

File: gui/src/i18n/de/translation.json [hunk 10]
```diff
@@ -0,0 +1,462 @@
...
+      "back": "Zurück zu den WLAN-Anmeldedaten",
+      "title": "Tracker zuweisen",
+      "description": "Wählen sie nun aus, welcher Tracker wo hin geht. Klicken Sie auf einen Ort, an dem Sie einen Tracker platzieren möchten",
+      "assigned": "{{assigned}} von {{trackers}} Trackern zugewiesen",
+      "advanced": "Erweiterte Zuweisungspositionen anzeigen",
+      "next": "Ich habe alle Tracker zugewiesen"
+    },
+    "manual-mounting": {
+      "back": "Zurück zum Eintritt in die VR",
+      "title": "Manuelles Befestigen",
```
Comment (unlogisch04) [inline gui/src/i18n/de/translation.json:367]: Not sire if too long. Suggestions welcome
```suggestion
      "title": "Manuelle definiton der Befestigungsposition",
```

**Review by unlogisch04 — COMMENTED** [review]
Not all done

File: gui/src/i18n/de/translation.json [hunk 11]
```diff
@@ -0,0 +1,462 @@
...
+    "open-wifi": "Mit WLAN verbinden",
+    "open-serial": "Seriellen Konsole öffnen",
+    "submit": "Absenden!",
+    "close": "Schließen"
+  },
+  "navbar": {
+    "home": "Start",
+    "body-proportions": "Körpermaße",
+    "trackers-assign": "Tracker-Positionen",
+    "mounting": "Befestigungs-Kalibrierung",
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:73]: The Translation of this word has been especially difficult in the Assistant part. As it is mentioned multiple times and should be understandable by beginners

File: gui/src/i18n/de/translation.json [hunk 12]
```diff
@@ -0,0 +1,462 @@
...
+    "onboarding": "Setup-Assistent",
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:74]: maybe "Einrichtungs-Assistent"? 
also the UI does line breaks on dashes, so it won't be all in one line

File: gui/src/i18n/de/translation.json [hunk 13]
```diff
@@ -0,0 +1,462 @@
...
+        "description.p1": "Vorhersage prognostiziert Bewegung, während Glättung Bewegung glättet.",
+        "filtering-type": {
+          "title": "Filter-Typ",
+          "none": {
+            "label": "Kein Filter",
+            "description": "Verwenden Sie Rotationen unverändert. Kein Filter wird durchgeführt."
+          },
+          "smoothing": {
+            "label": "Glättung",
+            "description": "Glättet Bewegungen, fügt aber etwas Latenz hinzu."
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:196]: changed to "Verzögerung"

File: gui/src/i18n/de/translation.json [hunk 14]
```diff
@@ -0,0 +1,462 @@
...
+        "subtitle": "Double Tap Quick Reset",
+        "description": "Aktivieren oder deaktivieren Sie Double Tap Quick Reset. Wenn es aktiviert ist, wird das Doppeltippen irgendwo auf dem höchsten Brusttracker einen schnellen Reset auslösen. Die Verzögerung ist die Zeit zwischen dem Registrieren eines Taps und dem Zurücksetzen.",
+        "enable": "Double tap quick reset",
+        "delay": "Verzögerung"
+      },
+      "interface": {
+        "title": "Schnittstelle",
+        "dev-mode": {
+          "title": "Entwicklermodus",
+          "description": "Dieser Modus kann nützlich sein, wenn Sie tiefgreifende Daten benötigen oder auf einer fortgeschrittenen Ebene mit verbundenen Trackern interagieren möchten",
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:240]: Sounds good, maybe also add a warning like: "Kann unter Umständen oder bei falscher Benutzung die Tracking Genauigkeit verschlechtern oder zu anderen Problemen führen."?

File: gui/src/i18n/de/translation.json [hunk 15]
```diff
@@ -0,0 +1,462 @@
...
+          "label": "Entwicklermodus"
+        },
+        "serial-detection": {
+          "title": "Serielle Geräteerkennung",
+          "description": "Diese Option zeigt jedes Mal, wenn Sie ein neues serielles Gerät anschließen, das ein Tracker sein könnte, ein Pop-up-Fenster an. Es hilft bei der Verbesserung des Einrichtungsprozesses eines Trackers",
+          "label": "Serielle Geräteerkennung"
+        },
+        "lang": {
+          "title": "Wählen Sie eine Sprache",
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:249]: changed to "Sprachauswahl"

File: gui/src/i18n/de/translation.json (hunk 10 above)
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:367]: In which window was this? Didn't find it

File: gui/src/i18n/de/translation.json [hunk 16]
```diff
@@ -0,0 +1,462 @@
...
+  },
+  "body-part": {
+    "NONE": "Nicht zugewiesen",
+    "HEAD": "Kopf",
+    "NECK": "Hals",
+    "RIGHT_SHOULDER": "Rechte Schulter",
+    "RIGHT_UPPER_ARM": "Rechter Oberarm",
+    "RIGHT_LOWER_ARM": "Rechter Unterarm",
+    "RIGHT_HAND": "Rechte Hand",
+    "RIGHT_UPPER_LEG": "Rechtes oberes Bein",
```
[the remark this answers is not in the document]
Comment (unlogisch04) [reply gui/src/i18n/de/translation.json:18]: > Wurde gemacht um mehr der englischen Übersetzung zu entsprechen. Zusätzlich ist Unterschenkel nicht korrekt, da Schenkel nur bis zum Knie geht

Laut Informationen vom Duden ist Unterschenkel korrekt. 
https://www.duden.de/rechtschreibung/Unterschenkel
"Teil des Beines zwischen Knie und Fuß"

File: gui/src/i18n/de/translation.json (hunk 14 above)
[the remark this answers is not in the document]
Comment (unlogisch04) [reply gui/src/i18n/de/translation.json:240]: As it is not in the original, i would not add it. Also, the development mode ifself does not change the tracking. You will have more options that can change it.

File: gui/src/i18n/de/translation.json (hunk 16 above)
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:18]: Okay, indemfall ändere ich alle auf ober/unter Schenkel ab

File: gui/src/i18n/de/translation.json (hunk 14 above)
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:240]: changed to your suggestion

File: gui/src/i18n/de/translation.json (hunk 10 above)
Comment (Philip6294) [reply gui/src/i18n/de/translation.json:367]: Updated it, let's just hope that it works out

**Comment by ImUrX** [comment]
btw the error is just because you updated ``solarxr-protocol``'s commit :p

**Comment by Philip6294** [comment]
ah okay, well the solarxr Submodule is causing some problems for me

**Comment by ImUrX** [comment]
yeah, gimme a moment I will fix it :p

File: gui/public/i18n/de/translation.ftl [hunk 17]
```diff
@@ -0,0 +1,520 @@
...
+onboarding-manual_proportions-auto = Automatische Kalibrierung
+
+## Tracker automatic proportions setup
+onboarding-automatic_proportions-back = Gehen Sie zurück zum Reset-Tutorial
+onboarding-automatic_proportions-title = Messen Sie Ihre Proportionen
+onboarding-automatic_proportions-description = Damit die SlimeVR-Tracker funktionieren, müssen wir Ihre Proportionen kennen. Diese kurze Kalibrierung wird sie für Sie messen.
+onboarding-automatic_proportions-manual = Manuelle Kalibrierung
+onboarding-automatic_proportions-prev_step = Vorheriger Schritt
+onboarding-automatic_proportions-put_trackers_on-title = Ziehen sie ihre Tracker an
+onboarding-automatic_proportions-put_trackers_on-description = Um Ihre Proportionen zu kalibrieren, werden wir die Tracker verwenden, die Sie gerade zugewiesen haben. Legen Sie alle Ihre Tracker an, Sie können sehen, welche welche sind in der Abbildung rechts.
```
Comment (TheDevMinerTV) [inline gui/public/i18n/de/translation.ftl:487]:
```suggestion
onboarding-automatic_proportions-put_trackers_on-description = Um Ihre Proportionen zu kalibrieren, werden wir die Tracker verwenden, die Sie gerade zugewiesen haben. Legen Sie alle Ihre Tracker an. Sie können rechts in der Abbildung sehen, welche welche sind.
```

File: gui/public/i18n/de/translation.ftl [hunk 18]
```diff
@@ -0,0 +1,520 @@
...
+onboarding-automatic_proportions-put_trackers_on-next = Ich habe alle meine Tracker an
```
Comment (TheDevMinerTV) [inline gui/public/i18n/de/translation.ftl:488]:
```suggestion
onboarding-automatic_proportions-put_trackers_on-next = Ich habe alle meine Tracker angelegt
```

File: gui/public/i18n/de/translation.ftl [hunk 19]
```diff
@@ -0,0 +1,520 @@
...
+onboarding-automatic_proportions-preparation-title = Vorbereitung
+onboarding-automatic_proportions-preparation-description = Stellen Sie einen Stuhl direkt hinter Ihnen in Ihrem Spielbereich. Seien Sie bereit, während der Autobone-Einrichtung zu sitzen.
+onboarding-automatic_proportions-preparation-next = Ich stehe vor einem Stuhl
+onboarding-automatic_proportions-start_recording-title = Bereite dich auf Bewegung vor
+onboarding-automatic_proportions-start_recording-description = Wir werden nun einige bestimmte Posen und Bewegungen aufnehmen. Diese werden im nächsten Bildschirm angezeigt. Bereite dich darauf vor, wenn der Knopf gedrückt wird zu starten!
+onboarding-automatic_proportions-start_recording-next = Aufnahme starten
+onboarding-automatic_proportions-recording-title = Aufnahme
+onboarding-automatic_proportions-recording-description-p0 = Aufnahme läuft...
+onboarding-automatic_proportions-recording-description-p1 = Mache die unten beschriebenen Bewegungen:
```
Comment (TheDevMinerTV) [inline gui/public/i18n/de/translation.ftl:497]:
```suggestion
onboarding-automatic_proportions-recording-description-p1 = Machen Sie die unten beschriebenen Bewegungen:
```

File: gui/public/i18n/de/translation.ftl [hunk 20]
```diff
@@ -0,0 +1,520 @@
...
+onboarding-automatic_proportions-description = Damit die SlimeVR-Tracker funktionieren, müssen wir Ihre Proportionen kennen. Diese kurze Kalibrierung wird sie für Sie messen.
+onboarding-automatic_proportions-manual = Manuelle Kalibrierung
+onboarding-automatic_proportions-prev_step = Vorheriger Schritt
+onboarding-automatic_proportions-put_trackers_on-title = Ziehen sie ihre Tracker an
+onboarding-automatic_proportions-put_trackers_on-description = Um Ihre Proportionen zu kalibrieren, werden wir die Tracker verwenden, die Sie gerade zugewiesen haben. Legen Sie alle Ihre Tracker an, Sie können sehen, welche welche sind in der Abbildung rechts.
+onboarding-automatic_proportions-put_trackers_on-next = Ich habe alle meine Tracker an
+onboarding-automatic_proportions-preparation-title = Vorbereitung
+onboarding-automatic_proportions-preparation-description = Stellen Sie einen Stuhl direkt hinter Ihnen in Ihrem Spielbereich. Seien Sie bereit, während der Autobone-Einrichtung zu sitzen.
+onboarding-automatic_proportions-preparation-next = Ich stehe vor einem Stuhl
+onboarding-automatic_proportions-start_recording-title = Bereite dich auf Bewegung vor
```
Comment (TheDevMinerTV) [inline gui/public/i18n/de/translation.ftl:492]:
```suggestion
onboarding-automatic_proportions-start_recording-title = Bereiten Sie sich auf ein paar Bewegungen vor
```

File: gui/public/i18n/de/translation.ftl [hunk 21]
```diff
@@ -0,0 +1,520 @@
...
+onboarding-automatic_proportions-start_recording-description = Wir werden nun einige bestimmte Posen und Bewegungen aufnehmen. Diese werden im nächsten Bildschirm angezeigt. Bereite dich darauf vor, wenn der Knopf gedrückt wird zu starten!
```
Comment (TheDevMinerTV) [inline gui/public/i18n/de/translation.ftl:493]:
```suggestion
onboarding-automatic_proportions-start_recording-description = Wir werden nun einige bestimmte Posen und Bewegungen aufnehmen. Diese werden im nächsten Bildschirm angezeigt. Bereiten Sie sicht darauf vor, wenn Sie den Knopf drücken!
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: gui/public/i18n/de/translation.ftl [hunk 22]
```diff
@@ -0,0 +1,520 @@
...
+onboarding-continue = Fortsetzen
+onboarding-wip = Noch in Bearbeitung
+
+## WiFi setup
+onboarding-wifi_creds-back = Zurück zur Einführung
+onboarding-wifi_creds = WLAN-Zugangsdaten eingeben
+# This cares about multilines
+onboarding-wifi_creds-description =
+    Die Tracker nutzen diese Zugangsdaten, um sich mit dem WLAN zu verbinden
+    Bitte verwenden Sie die Zugangsdaten, mit denen ihr PC gerade verbunden sind
```
[the remark this answers is not in the document]
Comment (Philip6294) [reply gui/public/i18n/de/translation.ftl:375]: are those ´´´ intentional?

[the remark this answers is not in the document]
Comment (TheDevMinerTV) [reply gui/public/i18n/de/translation.ftl:374]: Fixed

**Review by TheDevMinerTV — APPROVED** [review]
LGTM

**Review by Eirenliel — APPROVED** [review]

**Comment by Eirenliel** [comment]
Y'all fucking finally did it. Good job. (inb4 fixes next day)

**Pull request merged**